ahemap/templates/base.html

```diff
@@ -87,6 +87,18 @@
     <script type="text/javascript" src="https://search.sites.columbia.edu/cu-privacy-notice/cu-privacy-notice.js"></script>
 </head>
 
+
+<!-- Google tag (gtag.js) -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=G-8K69HX6VFF"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+
+  gtag('config', 'G-8K69HX6VFF');
```

Comment:
We usually keep the analytics tag in salt_piller under our apps.sls file. You would need to change the current ahemap analytics tag to this one and reference it like so:

`    {% if GA_CODE %}
    <!-- Global site tag (gtag.js) - Google Analytics -->
    <script async src="https://www.googletagmanager.com/gtag/js?id={{GA_CODE}}"></script>
    <script>
          window.dataLayer = window.dataLayer || [];
          function gtag(){dataLayer.push(arguments);}
          gtag('js', new Date());
           gtag('config', '{{GA_CODE}}', { 'anonymize_ip': true });
    </script>
    {% endif %}`

Reply:
Also noting that the old google analytics code is still around down on line 188